.github/workflows/manual-test-matrix-workflow.yaml

```diff
@@ -51,7 +51,7 @@ jobs:
         uses: actions/setup-java@v3
         with:
           distribution: "adopt"
-          java-version: "17"
+          java-version: "11"
```

Comment:
This change can be probably reverted as the related issue is fixed upstream.

Reply:
the issue is fixed. Java 17 used with the latest keycloak